lab-guides/01-AzureLocalCloudDeployment/readme.md

```diff
@@ -58,16 +58,16 @@ Dell only supports Dell ISO that includes Dell drivers.
 
 * Make sure you hydrate latest Azure Local Image using CreateParentDisk.ps1 located in ParentDisks folder
 
-* Note: this lab uses ~60GB RAM. To reduce amount of RAM, you would need to reduce number of nodes.
+* Note: this lab uses ~85GB RAM. To reduce amount of RAM, you would need to reduce number of nodes.
```

Comment:
done a test with real conditions and including host OS WS 2025 the RAM consumption sits around 85 GB RAM
<img width="758" height="226" alt="image" src="https://github.com/user-attachments/assets/ccdaa775-0f04-4d52-b79f-54e65de84d7f" />